app/enterprise/0.33-x/admin-gui/configuration/authentication.md

```diff
@@ -48,6 +48,8 @@ HTTP/1.1 201 Created
 }
 ```
 
+Save the generated `user_token` (`a3cebf9e-820d-4543-b760-2b6986e3bb9d` in the example above) for use later. It must be sent in the `Kong-Admin-Token` header when RBAC is enabled if using a client (e.g. cURL or HTTPie) other than the Admin GUI.
```

Comment:
Might want to add caveat that if you are using a client that is able to access `:8001` (non-proxied admin api) then it is not required.

Reply:
I am referring to the credential. My comment was was meant to be on the entire PR so referring to where he says they are distinct.